Sources/Sentry/SentryProfiler.mm

```diff
@@ -204,7 +204,7 @@ - (void)start
                 }
                 [samples addObject:sample];
             },
-            100 /** Sample 100 times per second */);
+            101 /** Sample 101 times per second */);
```

Comment:
@armcknight, it would be awesome to add the info from the PR as a comment so we know why it's 101.

We simply change the profiler's internal sampling rate from 100 to 101 (a prime number).
This reduces the potential for [lock-step](https://stackoverflow.com/a/45471031) problems. With a composite number, it's possible for repeated subroutines with durations that are divisors of the sample rate to occur in phase with the sampler's starting and/or stopping, which can produce misleading results.